contributing_to_docs/doc_guidelines.adoc

```diff
@@ -257,6 +257,31 @@ link:https://redhat-documentation.github.io/modular-docs/#creating-procedure-mod
 When needed, use `.Prerequisites`, `.Next steps`, or `.Additional resources` syntax to suppress TOC formatting within a module. Do not use `==` headings or xrefs in modules.
 ====
 
+[id="using-conscious-language"]
+== Using conscious language
+
+To assist with the removal of the problematic word "master" from the documentation, use the following terminology when referring to OpenShift control plane nodes:
+
+[options="header"]
+|===
+|Branch |Control plane node reference
+
+|`main` and `enterprise-4.9`
+|Control plane node
+
+|`enterprise-4.8` and earlier
+|Control plane (also known as master) node
```

Comment:
So in looking in the repo, it looks like @vikram-redhat's sweep actually worded it like this:

```
control plane node (also known as the master node)
```
However I do actually like your wording better. It's more concise/less repetitive, and would use the same wording no matter whether it was an "a" or "the" in front, or if it's plural or singular. @vikram-redhat does it have to be worded the way you did it (repeating "node" or "host", etc.), or can we use what @pneedle-rh has here?

Reply:
This wording is indeed better. Please go with this.

Reply:
Thank you.

Reply:
I am going to create a script to update "control plane node (also known as the master node)" in the docs to "Control plane (also known as master) node", so that it is in-line with these guidelines. I will create a separate series of PRs for enterprise-4.8 and earlier for that.